test/test_files/tinysnb/function/cast.test

```diff
@@ -424,6 +444,16 @@ Hubert Blaine Wolfeschlegelsteinhausenbergerdorff
 [77,64,100,54]
 
 
+-LOG CastFixedListToFixedList
+-STATEMENT MATCH(p:person) where p.ID > 4 RETURN cast(p.grades, "UINT64[]"), cast(p.grades, "UINT32[]"), cast(p.grades, "UINT16[]"), cast(p.grades, "UINT8[]"), cast(p.grades, "INT8[]"), cast(p.grades, "INT16[]"), cast(p.grades, "INT32[]"), cast(p.grades, "INT64[]"), cast(p.grades, "INT128[]"), cast(p.grades, "STRING[]"), cast(p.grades, "DOUBLE[]"), cast(p.grades, "FLOAT[]");
+---- 6
+[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43,83,67,43]|[43.000000,83.000000,67.000000,43.000000]|[43.000000,83.000000,67.000000,43.000000]
+[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76,88,99,89]|[76.000000,88.000000,99.000000,89.000000]|[76.000000,88.000000,99.000000,89.000000]
+[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77,64,100,54]|[77.000000,64.000000,100.000000,54.000000]|[77.000000,64.000000,100.000000,54.000000]
+[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80,78,34,83]|[80.000000,78.000000,34.000000,83.000000]|[80.000000,78.000000,34.000000,83.000000]
+[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96,59,65,88]|[96.000000,59.000000,65.000000,88.000000]|[96.000000,59.000000,65.000000,88.000000]
+|||||||||||
```

Comment:
I am a little confused about this test, we only have 5 nodes with ID > 4, why do you get 6 nodes?

Reply:
`-LOG InsertToPerson
-STATEMENT CREATE (a:person {ID: 12})
---- ok`
A node was created with ID 12